### PR TITLE
chore(deps): Upgrade @box/frontend and associated dependencies

### PR DESCRIPTION
**Pull request by jstoffan**
I'm hesitant to fix the `import/no-cycle` issues in the `HotkeyLayer` components, as I think they're heavily used in many of our apps. It also looks like the `AvatarDetails` component is duplicated, if @box/collab could take a look after this is merged.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jstoffan has signed the CLA. Thanks for the pull request!

File: src/api/Folder.js [hunk 1]
```diff
@@ -217,13 +213,14 @@ class Folder extends Item {
         this.errorCode = ERROR_CODE_FETCH_FOLDER;
         let params = { fields: requestFields.toString() };
         if (!noPagination) {
-            params = Object.assign({}, params, {
+            params = {
```
Comment (tjgupta) [inline src/api/Folder.js]: LOL, this seems harder to understand than before. Not to start an argument over the eslint rules, but... 🤷‍♂

File: src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js [hunk 2]
```diff
@@ -34,6 +34,8 @@ const Datestamp = ({ date }: { date: ISODate | Date }) => {
     return <ReadableTime timestamp={new Date(date).getTime()} alwaysShowTime relativeThreshold={0} />;
 };
 
+// TODO: Move to a separate component with its own Props type
+/* eslint-disable react/prop-types */
 const AvatarDetails = React.memo(({ user, status, completedAt, className }) => {
```
Comment (alexkrolick) [inline src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]:
```suggestion
type DetailsProps = {
    user: { name: string },
    status: $Keys<statusMessages>,
    completedAt?: ?Date,
    className?: string,
};
const AvatarDetails = React.memo(({ user, status, completedAt, className }: DetailsProps) =>
```

File: src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js [hunk 3]
```diff
@@ -34,6 +34,8 @@ const Datestamp = ({ date }: { date: ISODate | Date }) => {
     return <ReadableTime timestamp={new Date(date).getTime()} alwaysShowTime relativeThreshold={0} />;
 };
 
+// TODO: Move to a separate component with its own Props type
```
Comment (alexkrolick) [inline src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]:
```suggestion
```

File: src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js [hunk 4]
```diff
@@ -34,6 +34,8 @@ const Datestamp = ({ date }: { date: ISODate | Date }) => {
...
+/* eslint-disable react/prop-types */
```
Comment (alexkrolick) [inline src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]:
```suggestion
```

File: src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js [hunk 5]
```diff
@@ -47,6 +49,7 @@ const AvatarDetails = React.memo(({ user, status, completedAt, className }) => {
         </div>
     );
 });
+/* eslint-enable react/prop-types */
```
Comment (alexkrolick) [inline src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]:
```suggestion
```

File: src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js (hunk 2 above)
Comment (jstoffan) [reply src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]: The above is going to throw errors due to out-of-order prop keys, I think. Can we handle this cleanup in a follow-up?

Comment (jstoffan) [reply src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]: The above is going to throw errors due to out-of-order prop keys, I think. Can we handle this cleanup in a follow-up?

Comment (alexkrolick) [reply src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.js]: Yes, if you want.

File: src/api/Folder.js (hunk 1 above)
Comment (jstoffan) [reply src/api/Folder.js]: It's the same logic.

**Pull request merged**